src/index.ts

```diff
@@ -47,14 +47,21 @@ const handleMessage = (message: any) => {
       if (message.state === Connection.CONNECTED) {
         machine.send({ type: 'NEW_BLOCK' });
       }
+      if (message.state === Connection.CONNECTING) {
+        logger.error(`Websocket is attempting to connect to ${process.env.DEFAULT_SERVER}`);
+      }
+      if (message.state === Connection.CLOSED) {
+        logger.error('Websocket connection was closed.');
```

Comment:
Here I think I agree with the error. We expect the websocket connection to be always opened with the full node